packages/ui-stencil/src/components/internal/orama-modal/orama-modal.scss

```diff
@@ -45,3 +45,31 @@
 .modal-title {
   @include screen-reader-only();
 }
+
+.embed {
```

Comment:
I think we no longer need this, can be removed

Reply:
we still have the modal title in the component, I will leave it for now.

```
 <h1 id="modalTitle" class="modal-title">
   {this.mainTitle}
 </h1>
```

Reply:
Yes I was refering to .embed class only